src/_tutorials/web/low-level-html/connect-dart-html.md

```diff
@@ -370,7 +398,7 @@ on how to convert a DartPad app into one you can run in the browser.
 Here's the full HTML code for this app,
 assuming that the Dart code is in a file named `main.dart`:
 
-```dart
+{% prettify html tag=pre+code %}
```

Comment:
Good catch!

Does  <code>```html</code> not work?

Reply:
It does, but the resulting HTML highlighting does not seem to handle it well at all:

<img width="952" alt="image" src="https://user-images.githubusercontent.com/18372958/142935889-6f8b7aab-3de3-4247-983d-26acfb9aeeac.png">

For now, I'll stick to prettify for these and investigate this separately (https://github.com/dart-lang/site-www/issues/3685)

Reply:
Ah right. I was encountering this myself recently.